src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 8px;
+}
+
+.big-search {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.search {
+  width: 100%;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  outline: none;
+  border: 1px solid #e1e7ed;
+
+}
+
+/* Styles for big input */
+
+.input-big-search {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background: url(images/Search.svg) no-repeat 26px / 19px;
+  padding: 23px 20px 23px 62px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: 300;
+  border-radius: 4px;
+}
+
+.input-big-search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input-big-search:focus {
+  font-weight: 900;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background: url(images/Search.svg) no-repeat 25px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.input-big-search:focus::placeholder {
+  font-weight: 300;
+  text-shadow: none;
+  filter: none;
+}
+
+/* Styles for small input */
+
+.input-small-search {
+  font-size: 14px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
move this rule to the common `search` class

Reply:
I moved box-shadow in the common search class. 
But font-size is different for small search (font-size: 14px) and big search (font-size: 16px), so I left it as it is.